core/domain/rights_manager.py

```diff
@@ -295,26 +436,60 @@ def get_collection_rights(collection_id, strict=True):
 
 
 def get_collection_owner_names(collection_id):
-    """Retrieves the owners for this collection from the datastore."""
+    """Retrieves the owners for this collection from the datastore.
+
+    Args:
+        collection_id: str. ID of collection.
+
+    Returns:
+        list. ID of owners for this collection.
```

Comment:
list(str). Human-readable usernames (or truncated email addresses) of owners for this collection.

(I determined the above by looking at the docstring for get_human_readable_user_ids.)